hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/ReadReplicas.java

```diff
@@ -57,11 +64,12 @@
  * given key. It also generates a manifest file with information about the
  * downloaded replicas.
  */
-@CommandLine.Command(name = "read-replicas",
+@CommandLine.Command(name = "checksums",
     description = "Reads every replica for all the blocks associated with a " +
-        "given key.")
+        "given key.",
+    aliases = {"read-replicas"})
 @MetaInfServices(DebugSubcommand.class)
-public class ReadReplicas extends KeyHandler implements DebugSubcommand {
+public class ReadReplicas extends Handler implements DebugSubcommand {
```

Comment:
Suggest to rename this class/file name as well to match the command name.

Reply:
Implementing `DebugSubcommand` adds this one directly under `ozone debug`, so it should be removed if we want to move, not copy.

```suggestion
public class ReadReplicas extends Handler {
```

Note that acceptance tests exercise this tool, they will need to be updated:

```
hadoop-ozone/dist/src/main/smoketest/debug/ozone-debug.robot
hadoop-ozone/dist/src/main/smoketest/debug/ozone-debug-tests.robot
hadoop-ozone/dist/src/main/smoketest/debug/ozone-debug-stale-datanode.robot
hadoop-ozone/dist/src/main/smoketest/debug/ozone-debug-corrupt-block.robot
hadoop-ozone/dist/src/main/smoketest/debug/ozone-debug-dead-datanode.robot
```

Reply:
Done